hyco-https/lib/HybridConnectionHttpsServer.js

```diff
@@ -573,52 +576,60 @@ function controlChannelRequest(server, message) {
   // handler to call when the connection sequence completes
   var self = server;
   
-  if ( message.request.method) {
+  if (message.request.method) {
```

Comment:
>message.request.method [](start = 6, length = 22)

Is is correct that this 'if' statement gets false if a rendezvous is required to read the request?

Can we add a test which does large request + (any) response?

Reply:
The "if" statement works, but it's probably better to be more explicit

---
In reply to: [315884261](https://github.com/Azure/azure-relay-node/pull/33#discussion_r315884261) [](ancestors = 315884261)

Reply:
request.body is tri-state. null | true | false. I don't think this test is quite right.

I wasn't actually pushing for any code change (maybe just a comment saying `// method is not present if a rendezvous is required to read the request message` )

---
In reply to: [315951263](https://github.com/Azure/azure-relay-node/pull/33#discussion_r315951263) [](ancestors = 315951263,315884261)

Reply:
I just read the protocol documentation again. If rendezvous is required then the `{ request }` json has nothing other than `address`. Therefore the original `if (message.request.method)` is likely good enough (if my understanding of JS is correct). I simply added this comment for us to double check this assumption was correct.

---
In reply to: [316284315](https://github.com/Azure/azure-relay-node/pull/33#discussion_r316284315) [](ancestors = 316284315,315951263,315884261)